docs/source/core.rst

```diff
@@ -66,51 +66,141 @@ useBlock
 useBlockMeta
 ============
 
+useChainCall
+============
+
+Makes a call to a specific contract and returns value. Hook will cause component to refesh whenever new block is mined and value is changed.
+
+Calls will be combined into a single multicall across all uses of *useChainCall* and *useChainCalls*.
+
+It is recommended to use `useContractCall`_ where applicable instead of this method.
+
+*Parameters*
+
+- ``call: ChainCall | Falsy`` - a single call, also see `ChainCall`_. A call can be `Falsy`, as it is important to keep the same ordering of hooks even if in given render cycle there might be not enough information to perfom a call.
+
+
 useChainCalls
 =============
 
+Makes multiple call to a specific contracts and returns values. Hook will cause component to refesh when values change.
+
+Calls will be combined into a single multicall across all uses of *useChainCall* and *useChainCalls*.
+It is recommended to use `useContractCall`_ where applicable instead of this methods.
+
 *Parameters*
 
-- ``calls: ChainCall[]``
+- ``calls: ChainCall[]`` - list of calls, also see `ChainCall`_. Calls need to be in the same order accros component renders.
+
+useContractCall
+===============
+Makes a call to a specific contract and returns value. Hook will cause component to refesh when new block is mined and return value changes.
+A syntax sugar for `useChainCall`_ that uses ABI, function name and arguments instead of raw data.
+
+**Parameters**
+
+- ``calls: ContractCall | Falsy`` - a single call to a contract , also see `ContractCall`_
+
+**Returns**
+
+- ``any[] | undefined`` - result of a call or undefined if call didn't return yet
+
+useContractCalls
+===============
+Makes calls to specific contracts and returns values. Hook will cause component to refesh when new block is mined and any of return values changes.
+A syntax sugar for `useChainCalls`_ that uses ABI, function name and arguments instead of raw data.
```

Comment:
```suggestion
Makes calls to specific contracts and returns values. The hook will cause the component to refresh when a new block is mined and the return values change.
A syntax sugar for `useChainCalls`_ that uses ABI, function name, and arguments instead of raw data.
```